dashboard/test/models/script_test.rb

```diff
@@ -937,4 +937,37 @@ def populate_cache_and_disconnect_db
 
     assert_match Regexp.new(new_dsl_regex), ScriptDSL.serialize_to_string(script_copy)
   end
+
+  def has_hidden_script?(response)
+    response.any?(&:hidden)
+  end
```

Comment:
Make this private, at the bottom of the file.

Reply:
And I think `scripts` would be a more clear param name, especially since that's what's supplied in each call.